_build/data/InnoSetup.template

```diff
@@ -61,7 +61,7 @@ begin
         Sleep(500)
       end;
       Success := RenameFile(Src, Dest)
-      Inc(i)
```

Comment:
[Inc()](https://www.freepascal.org/docs-html/rtl/system/inc.html) is a function of the Pascal scripting used by/for InnoSetup scripts. Did you see any error with this script?

Reply:
hmm, i can't replicate the problem :man_shrugging:

I deinstalled InnoSetup, made sure there are no .islu files left on my drives, installed InnoSetup 5.6.1 and run build script. No errors. The two .islu files exist and 'Inc' wasn't be marked as unknown identifier.